rewrite-maven/src/main/java/org/openrewrite/maven/internal/RawPom.java

```diff
@@ -510,9 +512,9 @@ private List<org.openrewrite.maven.tree.Plugin> mapPlugins(@Nullable List<Plugin
         if (rawPlugins != null) {
             plugins = new ArrayList<>(rawPlugins.size());
             for (Plugin rawPlugin : rawPlugins) {
-
+                String pluginGroupId = rawPlugin.getGroupId();
                 plugins.add(new org.openrewrite.maven.tree.Plugin(
-                        rawPlugin.getGroupId(),
+                        pluginGroupId == null ? PLUGIN_DEFAULT_GROUPID : pluginGroupId,
```

Comment:
I don’t see a way to set it anywhere else because the field is final `final` in the raw plugin, and the constructors are generated by Lombok.